ReportSimulationOutput/measure.rb

```diff
@@ -1693,11 +1693,13 @@ def get_report_variable_data_timeseries(key_values, variables, unit_conv, unit_a
   def get_report_variable_data_timeseries_key_values_and_units(timeseries_frequency, var)
     keys = []
     units = ''
-    @msgpackDataTimeseries['Cols'].each do |col|
-      next unless col['Variable'].end_with? ":#{var}"
+    if not @msgpackDataTimeseries.nil?
```

Comment:
Checking for nil is the actual fix. If you request output variables that don't exist, _and request no other timeseries outputs_, then E+ won't even create the timeseries output msgpack file.

Reply:
So to be clear, we are ignoring non-existent output variables and not throwing a warning or anything like that?

Reply:
No, a warning is always issued. Previously it would hit an error before it got to providing a warning.